cmd/loop/main.go

```diff
@@ -225,11 +225,24 @@ func parseAmt(text string) (btcutil.Amount, error) {
 }
 
 func logSwap(swap *looprpc.SwapStatus) {
-	fmt.Printf("%v %v %v %v - %v",
-		time.Unix(0, swap.LastUpdateTime).Format(time.RFC3339),
-		swap.Type, swap.State, btcutil.Amount(swap.Amt),
-		swap.HtlcAddress,
-	)
+	if swap.Type == looprpc.SwapType_LOOP_OUT {
+		fmt.Printf("%v %v %v %v - %v",
+			time.Unix(0, swap.LastUpdateTime).Format(time.RFC3339),
+			swap.Type, swap.State, btcutil.Amount(swap.Amt),
+			swap.HtlcAddressP2Wsh,
+		)
+	} else {
+		fmt.Printf("%v %v %v %v -",
+			time.Unix(0, swap.LastUpdateTime).Format(time.RFC3339),
+			swap.Type, swap.State, btcutil.Amount(swap.Amt))
+		if swap.HtlcAddressP2Wsh != "" {
+			fmt.Printf(" p2wsh: %v", swap.HtlcAddressP2Wsh)
```

Comment:
Same comment about consistency upper/lower case of the address type

Reply:
done